browser/iframe-browser.js

```diff
@@ -10,21 +10,31 @@ define(function (require, exports, module) {
     // by default we use vertical orientation
     var _orientation = VERTICAL_ORIENTATION;
 
+
     /*
-     * Publicly avaialble function used to create an iframe within the second-panel
-     * url: Takes one argument of a url to a blob for the _update() function to use 
-     *      - if undefined we will not fill the iframe with an src
+     * Publicly avaialble function used to create an empty iframe within the second-panel
      */
-    function browse(url) {
+    function init() {
         //Get current GUI layout
```

Comment:
You probably only want this called once, so you might want to add this to the start of this function:

```js
if(_panel) {
  return;
}
```